src/utils/filterTodos.ts

```diff
@@ -0,0 +1,24 @@
+import { FilterStatus } from '../types/FilterStatus';
+import { Todo } from '../types/Todo';
+
+export const getFilteredTodos = (
+  todos: Todo[],
+  selectedFilter: string,
+) => {
+  let todosCopy = [...todos];
+
+  if (selectedFilter) {
+    todosCopy = todosCopy.filter(({ completed }) => {
+      switch (selectedFilter) {
+        case FilterStatus.Active:
+          return !completed;
+        case FilterStatus.Completed:
+          return completed;
+        default:
+          return todosCopy;
```

Comment:
In the default case, you get an array of `todosCopy` arrays. And `filter` creates a new array so you don't need to create copy in this case